gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/resources/organizations/ReporterResource.java

```diff
@@ -0,0 +1,162 @@
+import jakarta.ws.rs.container.Suspended;
+import jakarta.ws.rs.core.Response;
+
+import java.util.Optional;
+
+/**
+ * @author Titouan COMPIEGNE (titouan.compiegne at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+public class ReporterResource extends AbstractResource {
+
+    private final ReporterServiceProxy reporterService;
+
+    private final OrganizationService organizationService;
+
+    @Inject
+    public ReporterResource(ReporterServiceProxy reporterService, OrganizationService domainService) {
+        this.reporterService = reporterService;
+        this.organizationService = domainService;
+    }
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(summary = "Get a reporter",
+            description = "User must have the DOMAIN_REPORTER[READ] permission on the specified domain " +
+                    "or DOMAIN_REPORTER[READ] permission on the specified environment " +
+                    "or DOMAIN_REPORTER[READ] permission on the specified organization")
+    @ApiResponses({
+            @ApiResponse(responseCode = "200", description = "Reporter successfully fetched",
+                    content = @Content(mediaType = "application/json",
+                            schema = @Schema(implementation = Reporter.class))),
+            @ApiResponse(responseCode = "500", description = "Internal server error")})
+    public void get(
+            @PathParam("organizationId") String organizationId,
+            @PathParam("reporterId") String reporterId,
+            @Suspended final AsyncResponse response) {
+
+        checkPermission(ReferenceType.ORGANIZATION, organizationId, Permission.ORGANIZATION_REPORTER, Acl.READ)
+                .andThen(organizationService.findById(organizationId)
+                        .onErrorResumeWith(Single.error(new OrganizationNotFoundException(organizationId)))
+                        .flatMapMaybe(irrelevant -> reporterService.findById(reporterId))
+                        .switchIfEmpty(Maybe.error(new ReporterNotFoundException(reporterId)))
+                        .map(reporter -> {
+                            if (!reporter.getReference().matches(ReferenceType.ORGANIZATION, organizationId)) {
+                                throw new BadRequestException("Reporter does not belong to organization");
+                            }
+                            return Response.ok(reporter.apiRepresentation(false)).build();
+                        }))
+                .subscribe(response::resume, response::resume);
+    }
+
+    @PUT
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(summary = "Update a reporter",
+            description = "User must have the DOMAIN_REPORTER[UPDATE] permission on the specified domain " +
+                    "or DOMAIN_REPORTER[UPDATE] permission on the specified environment " +
+                    "or DOMAIN_REPORTER[UPDATE] permission on the specified organization")
```

Comment:
**TODO:** permission shouldn't be related to the domain or env here as we are at Org level, this is OK into the code but this description needs to be changed